server/grpc/gogoreflection/serverreflection.go

```diff
@@ -226,8 +226,8 @@ func decompress(b []byte) ([]byte, error) {
 }
 
 func typeForName(name string) (reflect.Type, error) {
-	pt := getMessageType(name)
-	if pt == nil {
+	pt, err := getMessageType(name)
```

Comment:
This should be reverted. The function always return nil as an error, so it will never trigger.

Reply:
done, was thinking about this at dinner